### PR TITLE
Rover: adding AP_RPM

### PR DESCRIPTION
**Pull request by magicrub**
simple copy of rpm_sensor implementation from Plane.

### CODE REVIEW

File: APMrover2/Rover.h [hunk 1]
```diff
@@ -60,6 +60,7 @@
 #include <AP_ServoRelayEvents/AP_ServoRelayEvents.h>
 #include <AP_Stats/AP_Stats.h>                      // statistics library
 #include <AP_Terrain/AP_Terrain.h>
+#include <AP_RPM/AP_RPM.h>
```
Comment (rmackay9) [inline APMrover2/Rover.h]: a small thing but I've actually put the includes in alphabetical order in Rover..

**Comment by rmackay9** [comment]
A few things..

1. has the rpm sensor been tested and confirmed to be sending rpm messages to the ground station?
2. now that we have wheel-distance messages being sent it's fine that the wheel encoders don't send rpm if:
    - it's possible for users to easily see the wheel-encoder distance messages in the ground station so that they can check the wheel encoders are working
   - we remove the send-wheel-encoder function and any variables it uses

**Comment by magicrub** [comment]
after talking with @rmackay9 he says this is OK but now that we're not using send_wheel_encoder() then we should remove it as well as wheel_encoder_rpm[2] which is only used by it.

**Comment by magicrub** [comment]
done. Please check again @rmackay9

**Comment by rmackay9** [comment]
I'll just wait for @khancyr's response before merging

File: APMrover2/Rover.h (hunk 1 above)
Comment (magicrub) [reply APMrover2/Rover.h]: fixed

**Comment by magicrub** [comment]
Skype chat with @khancyr :
Me: Any objections to merging this?
@khancyr : no more objections go for it!

**Pull request merged**